tests/PHPUnit/Fixtures/ThreeVisitsWithCustomEvents.php

```diff
@@ -91,7 +92,7 @@ protected function trackEventWithoutUrl(PiwikTracker $vis)
     {
         $url = $vis->pageUrl;
         $vis->setUrl('');
-        self::checkResponse($vis->doTrackEvent('CategoryTriggersGoal here', 'This is an event without a URL'));
+        self::checkResponse($vis->doTrackEvent('CategoryTriggersGoal here', 'This is an event without a URL', $name = false, $value = 23));
```

Comment:
be good to also add a system test for this to see that it was attributed in Goals report?

Reply:
There should be revenue in the existing Goals.get report, is that what you were thinking of or a new test entirely?

Reply:
Sorry I missed that expected change. Thought I looked at them all but obviously didn't notice that one. all good.